core/lib/Cardano/Address/Script.hs

```diff
@@ -328,7 +406,15 @@ prettyErrValidateScript = \case
     Malformed ->
         "Parsing of the script failed. The script should be composed of nested \
         \lists, and the verification keys should be either encoded as bech32."
-
+    InvalidTimelocks ->
+        "The list inside a script must contain at most two timelock conditions \
+        \and they cannot be contradictory."
+    DuplicateXPubs ->
+        "Teh cosigners in a script template must assume an unique extended public key."
```

Comment:
Typos: `Teh` -> `The`, `an unique` -> `a unique`

Reply:
fixed